### ISSUE TITLE
import fiona fails libgdal.so.20: undefined symbol: _ZN6libdap3DDSC1EPNS_15BaseTypeFactoryERKSs

### ISSUE THREAD

**Issue by akrherz** [conda-forge/fiona-feedstock#115]
Issue: updated to latest fiona on linux64 python 3.6 and it fails to import with
```
$ python
Python 3.6.7 | packaged by conda-forge | (default, Nov 21 2018, 03:09:43) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import fiona
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/miniconda3/envs/prod/lib/python3.6/site-packages/fiona/__init__.py", line 83, in <module>
    from fiona.collection import BytesCollection, Collection
  File "/opt/miniconda3/envs/prod/lib/python3.6/site-packages/fiona/collection.py", line 9, in <module>
    from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
ImportError: /opt/miniconda3/envs/prod/lib/python3.6/site-packages/fiona/../../../libgdal.so.20: undefined symbol: _ZN6libdap3DDSC1EPNS_15BaseTypeFactoryERKSs
```

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /opt/miniconda3/envs/prod:
#
# Name                    Version                   Build  Channel
affine                    2.2.2                      py_0    conda-forge
alabaster                 0.7.12                     py_0    conda-forge
alembic                   0.9.9                      py_0    conda-forge
apiclient                 1.0.3                     <pip>
appdirs                   1.4.3                      py_1    conda-forge
asn1crypto                0.24.0                py36_1003    conda-forge
astroid                   2.1.0                 py36_1000    conda-forge
async_generator           1.10                       py_0    conda-forge
atk                       2.25.90           hf2eb9ee_1001    conda-forge
atomicwrites              1.2.1                      py_0    conda-forge
attrs                     18.2.0                     py_0    conda-forge
Automat                   0.7.0                     <pip>
automat                   0.7.0                      py_1    conda-forge
babel                     2.6.0                      py_1    conda-forge
backcall                  0.1.0                      py_0    conda-forge
beautifulsoup4            4.7.1                 py36_1001    conda-forge
black                     18.6b4                    <pip>
blas                      1.1                    openblas    conda-forge
bleach                    3.1.0                      py_0    conda-forge
blinker                   1.4                        py_1    conda-forge
bokeh                     1.0.4                 py36_1000    conda-forge
boost-cpp                 1.68.0            h11c811c_1000    conda-forge
boto3                     1.9.82                     py_0    conda-forge
botocore                  1.12.82                    py_0    conda-forge
bottleneck                1.2.1           py36h3010b51_1001    conda-forge
boxsdk                    1.5.5                     <pip>
bzip2                     1.0.6             h14c3975_1002    conda-forge
ca-certificates           2018.11.29           ha4d7672_0    conda-forge
cachetools                2.1.0                     <pip>
cairo                     1.14.12           h80bd089_1005    conda-forge
cartopy                   0.17.0          py36hd1c7146_1000    conda-forge
cdo                       1.9.5                h249571a_5    conda-forge
certifi                   2018.11.29            py36_1000    conda-forge
cf_units                  2.0.1           py36h3010b51_1002    conda-forge
cffi                      1.11.5          py36h9745a5d_1001    conda-forge
cftime                    1.0.3.4         py36h3010b51_1000    conda-forge
chardet                   3.0.4                 py36_1003    conda-forge
click                     7.0                        py_0    conda-forge
click-plugins             1.0.4                      py_0    conda-forge
cligj                     0.5.0                      py_0    conda-forge
cloudpickle               0.6.1                      py_0    conda-forge
codecov                   2.0.15                    <pip>
configurable-http-proxy   1.3.0                         0    conda-forge
consistent-hash-git       0.3                       <pip>
constantly                15.1.0                     py_0    conda-forge
coverage                  4.5.2           py36h14c3975_1000    conda-forge
coveralls                 1.5.1                      py_0    conda-forge
cryptography              2.3.1           py36hb7f436b_1000    conda-forge
cryptography-vectors      2.4.2                   py_1000    conda-forge
curl                      7.63.0            h646f8bb_1000    conda-forge
cycler                    0.10.0                     py_1    conda-forge
cython                    0.29.3           py36hf484d3e_0    conda-forge
cytoolz                   0.9.0.1         py36h14c3975_1001    conda-forge
dask                      1.0.0                      py_0    conda-forge
dask-core                 1.0.0                      py_0    conda-forge
dbus                      1.13.0            h4e0c4b3_1000    conda-forge
decorator                 4.3.0                      py_0    conda-forge
defusedxml                0.5.0                      py_1    conda-forge
descartes                 1.1.0                      py_2    conda-forge
distributed               1.25.2                py36_1000    conda-forge
docopt                    0.6.2                      py_1    conda-forge
docutils                  0.14                  py36_1001    conda-forge
dropbox                   9.2.0                      py_0    conda-forge
eccodes                   2.9.2             hbdf2b72_1001    conda-forge
ecmwf_grib                1.14.7                        2    conda-forge
entrypoints               0.3                   py36_1000    conda-forge
enum34                    1.1.6                     <pip>
ephem                     3.7.6.0         py36h14c3975_1001    conda-forge
esmf                      7.1.0r               h35eb876_3    conda-forge
expat                     2.2.5             hf484d3e_1002    conda-forge
feedgen                   0.7.0                     <pip>
fftw                      3.3.8             h14c3975_1001    conda-forge
fiona                     1.8.4           py36heb36068_1001    conda-forge
flake8                    3.6.0                 py36_1000    conda-forge
fontconfig                2.13.1            h2176d3f_1000    conda-forge
freetype                  2.9.1             h3cfcefd_1004    conda-forge
freexl                    1.0.5             h14c3975_1002    conda-forge
fribidi                   1.0.5             h14c3975_1000    conda-forge
future                    0.16.0                    <pip>
gdal                      2.3.2            py36hb00a9d7_0    conda-forge
gdata                     3.0.0                     <pip>
gdk-pixbuf                2.36.12           h4f1c04b_1001    conda-forge
geographiclib             1.49                       py_0    conda-forge
geojson                   2.4.1                      py_0    conda-forge
geopandas                 0.4.0                      py_1    conda-forge
geopy                     1.18.1                     py_0    conda-forge
geos                      3.6.2                hfc679d8_4    conda-forge
geotiff                   1.4.2             hfe6da40_1005    conda-forge
gettext                   0.19.8.1          h9745a5d_1001    conda-forge
giflib                    5.1.4             h14c3975_1001    conda-forge
glib                      2.56.2            had28632_1001    conda-forge
gmp                       6.1.2             hf484d3e_1000    conda-forge
gobject-introspection     1.56.1          py36h9e29830_1001    conda-forge
google-api-python-client  1.7.4                     <pip>
google-auth               1.5.0                     <pip>
google-auth-httplib2      0.0.3                     <pip>
graphite2                 1.3.13            hf484d3e_1000    conda-forge
gsl                       2.2.1           blas_openblashddceaf2_6  [blas_openblas]  conda-forge
gst-plugins-base          1.12.5            h3865690_1000    conda-forge
gstreamer                 1.12.5            h0cc0488_1000    conda-forge
gtk2                      2.24.31           h5baeb44_1000    conda-forge
h5netcdf                  0.6.2                      py_0    conda-forge
h5py                      2.8.0           py36hfb8609e_1004    conda-forge
harfbuzz                  1.9.0             he243708_1001    conda-forge
hdf4                      4.2.13            h9a582f1_1002    conda-forge
hdf5                      1.10.3            hba1933b_1001    conda-forge
heapdict                  1.0.0                 py36_1000    conda-forge
html5lib                  1.0.1                      py_0    conda-forge
httplib2                  0.11.3                    <pip>
hyperlink                 17.3.1                     py_0    conda-forge
icu                       58.2              hf484d3e_1000    conda-forge
idna                      2.8                   py36_1000    conda-forge
iembot                    0.1.0                     <pip>
imagesize                 1.1.0                      py_0    conda-forge
incremental               17.5.0                     py_0    conda-forge
intel-openmp              2019.0                      118  
ipykernel                 5.1.0           py36h24bf2e0_1001    conda-forge
ipython                   7.2.0           py36h24bf2e0_1000    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
isort                     4.3.4                 py36_1000    conda-forge
jasper                    1.900.1           h07fcdf6_1005    conda-forge
jdcal                     1.4                        py_1    conda-forge
jedi                      0.13.2                py36_1000    conda-forge
jenks                     1.0                       <pip>
jinja2                    2.10                       py_1    conda-forge
jmespath                  0.9.3                      py_1    conda-forge
jpeg                      9c                h14c3975_1001    conda-forge
json-c                    0.12.1               h470a237_1    conda-forge
jsonschema                3.0.0a3               py36_1000    conda-forge
jupyter_client            5.2.4                      py_0    conda-forge
jupyter_core              4.4.0                      py_0    conda-forge
jupyterhub                0.9.4                 py36_1000    conda-forge
jupyterlab                0.35.4                   py36_0    conda-forge
jupyterlab_launcher       0.13.1                     py_2    conda-forge
jupyterlab_server         0.2.0                      py_0    conda-forge
kealib                    1.4.10            hc069b17_1000    conda-forge
kiwisolver                1.0.1           py36h6bb024c_1002    conda-forge
krb5                      1.16.3            hc83ff2d_1000    conda-forge
lazy-object-proxy         1.3.1           py36h14c3975_1000    conda-forge
libaec                    1.0.2             hf484d3e_1001    conda-forge
libcurl                   7.63.0            h01ee5af_1000    conda-forge
libdap4                   3.19.1            hd48c02d_1000    conda-forge
libedit                   3.1.20170329      hf8c457e_1001    conda-forge
libffi                    3.2.1             hf484d3e_1005    conda-forge
libgcc                    7.2.0                h69d50b8_2    conda-forge
libgcc-ng                 7.3.0                hdf63c60_0    conda-forge
libgdal                   2.3.2                hc73897a_0    conda-forge
libgfortran               3.0.0                         1    conda-forge
libgfortran-ng            7.2.0                hdf63c60_3    conda-forge
libiconv                  1.15              h14c3975_1004    conda-forge
libkml                    1.3.0             h328b03d_1009    conda-forge
libnetcdf                 4.6.1             hc6a9f4b_1010    conda-forge
libopenblas               0.2.20               h9ac9557_7  
libpng                    1.6.36            h84994c4_1000    conda-forge
libpq                     10.6              h13b8bad_1000    conda-forge
libprotobuf               3.6.1             hdbcaa40_1000    conda-forge
libsodium                 1.0.16            h14c3975_1001    conda-forge
libspatialindex           1.8.5             hf484d3e_1003    conda-forge
libspatialite             4.3.0a            h9968ff2_1023    conda-forge
libssh2                   1.8.0             h1ad7b7a_1003    conda-forge
libstdcxx-ng              7.3.0                hdf63c60_0    conda-forge
libtiff                   4.0.10            h648cc4a_1001    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libxcb                    1.13              h14c3975_1002    conda-forge
libxml2                   2.9.8             h143f9aa_1005    conda-forge
libxslt                   1.1.32            h4785a14_1002    conda-forge
locket                    0.2.0                      py_2    conda-forge
lxml                      4.3.0           py36h23eabaa_1000    conda-forge
lzo                       2.10              h14c3975_1000    conda-forge
mako                      1.0.7                      py_1    conda-forge
markupsafe                1.1.0           py36h14c3975_1000    conda-forge
matplotlib                3.0.2           py36h8a2030e_1001    conda-forge
matplotlib-base           3.0.2           py36h167e16e_1001    conda-forge
mccabe                    0.6.1                      py_1    conda-forge
metar                     1.7.0                     <pip>
metar                     1.6.0                     <pip>
metpy                     0.10.0                py36_1001    conda-forge
mistune                   0.8.4           py36h14c3975_1000    conda-forge
mkl                       2019.0                      118  
mkl_fft                   1.0.10                   py36_0    conda-forge
mkl_random                1.0.2                    py36_0    conda-forge
mock                      2.0.0                 py36_1000    conda-forge
mod-wsgi                  4.6.4                     <pip>
more-itertools            4.3.0                 py36_1000    conda-forge
mpi                       1.0                       mpich    conda-forge
mpich                     3.2.1             h1c2f66e_1007    conda-forge
msgpack-python            0.6.0           py36h6bb024c_1000    conda-forge
munch                     2.3.2                      py_0    conda-forge
nbconvert                 5.4.0                         0    conda-forge
nbformat                  4.4.0                      py_1    conda-forge
nco                       4.7.8                he007dc3_0    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
netcdf-fortran            4.4.4               h71ea97b_10    conda-forge
netcdf4                   1.4.2           py36h2b0cf2a_1000    conda-forge
nodejs                    11.7.0               hf484d3e_0    conda-forge
nose                      1.3.7                 py36_1002    conda-forge
notebook                  5.7.4                 py36_1000    conda-forge
numexpr                   2.6.9           py36h637b7d7_1000    conda-forge
numpy                     1.16.0          py36_blas_openblash1522bff_1000  [blas_openblas]  conda-forge
numpy-base                1.14.3           py36h0ea5e3f_1  
nwnserver                 0.2                       <pip>
oauth                     1.0.1                     <pip>
oauth2client              4.1.2                     <pip>
oauthlib                  2.1.0                      py_0    conda-forge
odfpy                     1.3.6                     <pip>
olefile                   0.46                       py_0    conda-forge
openblas                  0.3.3             h9ac9557_1001    conda-forge
openjpeg                  2.3.0             hf38bd82_1003    conda-forge
openssl                   1.0.2p            h14c3975_1002    conda-forge
ossuuid                   1.6.2             hf484d3e_1000    conda-forge
owslib                    0.17.1                     py_0    conda-forge
packaging                 18.0                       py_0    conda-forge
pamela                    1.0.0                      py_0    conda-forge
pandas                    0.23.4          py36h637b7d7_1000    conda-forge
pandoc                    2.5                           0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
pango                     1.40.14           hf0c64fd_1003    conda-forge
parso                     0.3.1                      py_0    conda-forge
partd                     0.3.9                      py_0    conda-forge
paste                     3.0.6                 py36_1000    conda-forge
patsy                     0.5.1                      py_0    conda-forge
pbr                       5.1.1                      py_0    conda-forge
pcre                      8.41              hf484d3e_1003    conda-forge
pep8                      1.7.1                      py_0    conda-forge
pexpect                   4.6.0                 py36_1000    conda-forge
pickleshare               0.7.5                 py36_1000    conda-forge
pika                      1.0.0b1                    py_0    conda-forge
pillow                    5.4.1           py36h00a061d_1000    conda-forge
pint                      0.9                        py_0    conda-forge
pip                       18.1                  py36_1000    conda-forge
pixman                    0.34.0            h14c3975_1003    conda-forge
pkginfo                   1.5.0.1                    py_0    conda-forge
pluggy                    0.8.1                      py_0    conda-forge
pockets                   0.6.2                     <pip>
pooch                     0.1+3.g5e21026            <pip>
pooch                     0.2.1                 py36_1000    conda-forge
pooch                     0.1a3                     <pip>
poppler                   0.67.0            h2fc8fa2_1002    conda-forge
poppler-data              0.4.9                         1    conda-forge
postgresql                10.6              h66cca7a_1000    conda-forge
proj4                     4.9.3                h470a237_8    conda-forge
prometheus_client         0.5.0                      py_0    conda-forge
prompt_toolkit            2.0.7                      py_0    conda-forge
protobuf                  3.6.1           py36hf484d3e_1001    conda-forge
psutil                    5.4.8           py36h14c3975_1000    conda-forge
psycopg2                  2.7.6.1         py36hb7f436b_1000    conda-forge
pthread-stubs             0.4               h14c3975_1001    conda-forge
ptyprocess                0.6.0                 py36_1000    conda-forge
py                        1.7.0                      py_0    conda-forge
pyasn1                    0.4.4                      py_1    conda-forge
pyasn1-modules            0.2.3                      py_0    conda-forge
pycodestyle               2.4.0                      py_1    conda-forge
pycparser                 2.19                       py_0    conda-forge
pycurl                    7.43.0.2         py36hb7f436b_0  
pyepsg                    0.4.0                      py_0    conda-forge
pyflakes                  2.0.0                      py_0    conda-forge
pygments                  2.3.1                      py_0    conda-forge
pygrib                    2.0.3           py36hf3f1b0b_1000    conda-forge
pyhamcrest                1.9.0                      py_2    conda-forge
pyIEM                     0.9.1                     <pip>
pyIEM                     0.10.0                    <pip>
pyIEM                     0.9.0                     <pip>
pyjwt                     1.7.1                      py_0    conda-forge
pykdtree                  1.3.1           py36h3010b51_1002    conda-forge
pyLDM                     0.1.0                     <pip>
pylint                    2.2.2                 py36_1000    conda-forge
pyopenssl                 18.0.0                py36_1000    conda-forge
pypandoc                  1.4                       <pip>
pyparsing                 2.3.1                      py_0    conda-forge
pyproj                    1.9.5.1         py36h2944ce7_1006    conda-forge
pyqt                      5.6.0           py36h13b7fb3_1008    conda-forge
pyrsistent                0.14.9          py36h14c3975_1000    conda-forge
PyRSS2Gen                 1.1                       <pip>
pysal                     1.14.4.post2          py36_1001    conda-forge
pyshp                     2.0.1                      py_0    conda-forge
pysocks                   1.6.8                 py36_1002    conda-forge
pytest                    4.1.1                 py36_1000    conda-forge
pytest-cov                2.6.1                      py_0    conda-forge
pytest-mpl                0.10                       py_0    conda-forge
pytest-runner             4.2                        py_1    conda-forge
python                    3.6.7             hd21baee_1001    conda-forge
python-dateutil           2.7.5                      py_0    conda-forge
python-editor             1.0.3                      py_0    conda-forge
python-memcached          1.59                       py_0    conda-forge
python-oauth2             1.0.1                    py36_0    conda-forge
python-twitter            3.4.2                     <pip>
pytz                      2018.9                     py_0    conda-forge
pyyaml                    3.13            py36h14c3975_1001    conda-forge
pyzmq                     17.1.2          py36h6afc9c9_1001    conda-forge
qt                        5.6.2             hf516382_1011    conda-forge
rasterio                  1.0.13           py36h54a03ab_0    conda-forge
rasterstats               0.13.0                     py_0    conda-forge
readline                  7.0               hf8c457e_1001    conda-forge
readme-renderer           24.0                      <pip>
reportlab                 3.5.12          py36hbd3ef63_1000    conda-forge
requests                  2.21.0                py36_1000    conda-forge
requests-oauthlib         1.0.0                      py_1    conda-forge
requests-toolbelt         0.8.0                      py_1    conda-forge
rrdtool                   0.1.14                    <pip>
rsa                       3.4.2                     <pip>
rtree                     0.8.3                 py36_1000    conda-forge
s3transfer                0.1.13                py36_1001    conda-forge
scikit-learn              0.19.2                    <pip>
scipy                     1.2.0           py36_blas_openblash1522bff_1200  [blas_openblas]  conda-forge
seaborn                   0.9.0                      py_0    conda-forge
send2trash                1.5.0                      py_0    conda-forge
service_identity          17.0.0                     py_0    conda-forge
setuptools                40.6.3                   py36_0    conda-forge
setuptools-markdown       0.2                       <pip>
shapely                   1.6.4           py36h0c48222_1000    conda-forge
simplegeneric             0.8.1                      py_1    conda-forge
simplejson                3.16.1           py36h470a237_0    conda-forge
sip                       4.18.1          py36hf484d3e_1000    conda-forge
siphon                    0.8.0                 py36_1001    conda-forge
six                       1.12.0                py36_1000    conda-forge
smartsheet-python-sdk     1.3.4.dev3+ga1559f8           <pip>
snowballstemmer           1.2.1                      py_1    conda-forge
snuggs                    1.4.1                      py_1    conda-forge
sortedcontainers          2.1.0                      py_0    conda-forge
soupsieve                 1.7.1                 py36_1000    conda-forge
sphinx                    1.8.3                 py36_1000    conda-forge
sphinxcontrib-napoleon    0.6.1                     <pip>
sphinxcontrib-websupport  1.1.0                      py_1    conda-forge
sqlalchemy                1.2.16          py36h14c3975_1000    conda-forge
sqlite                    3.26.0            h67949de_1000    conda-forge
statsmodels               0.9.0           py36h3010b51_1000    conda-forge
tblib                     1.3.2                      py_1    conda-forge
terminado                 0.8.1                 py36_1001    conda-forge
testpath                  0.4.2                 py36_1000    conda-forge
tk                        8.6.9             h84994c4_1000    conda-forge
tlslite                   0.4.9                     <pip>
toml                      0.9.4                     <pip>
toolz                     0.9.0                      py_1    conda-forge
tornado                   5.1.1           py36h14c3975_1000    conda-forge
tox                       3.4.0                     <pip>
tqdm                      4.29.1                     py_0    conda-forge
traitlets                 4.3.2                 py36_1000    conda-forge
twine                     1.12.1                     py_0    conda-forge
twisted                   18.9.0          py36h14c3975_1000    conda-forge
twittytwister             0.1.3                     <pip>
twython                   3.7.0                      py_0    conda-forge
txyam2                    0.5.1+weasyl.2            <pip>
typed-ast                 1.1.1           py36h14c3975_1000    conda-forge
tzcode                    2018g             h14c3975_1001    conda-forge
udunits2                  2.2.27.6          h4e0c4b3_1001    conda-forge
Unidecode                 1.0.22                    <pip>
unixodbc                  2.3.7             h227dcee_1000    conda-forge
uritemplate               3.0.0                      py_1    conda-forge
urllib3                   1.24.1                py36_1000    conda-forge
util-linux                2.21                          0  
verde                     0.0.1a0+72.g9ae9091           <pip>
virtualenv                16.0.0                    <pip>
wcwidth                   0.1.7                      py_1    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.32.3                   py36_0    conda-forge
windrose                  1.6                       <pip>
wordcloud                 1.5.0           py36h14c3975_1000    conda-forge
wrapt                     1.11.1           py36h14c3975_0    conda-forge
wrf-python                1.3.0           py36h6091dcd_1000    conda-forge
xarray                    0.11.2                py36_1000    conda-forge
xerces-c                  3.2.0                h5d6a6da_2    conda-forge
xlrd                      1.2.0                      py_0    conda-forge
xlsxwriter                1.1.2                      py_0    conda-forge
xlwt                      1.3.0                      py_1    conda-forge
xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
xorg-libice               1.0.9             h14c3975_1004    conda-forge
xorg-libsm                1.2.3             h4937e3b_1000    conda-forge
xorg-libx11               1.6.6             h14c3975_1000    conda-forge
xorg-libxau               1.0.8             h14c3975_1006    conda-forge
xorg-libxdmcp             1.1.2             h14c3975_1007    conda-forge
xorg-libxext              1.3.3             h14c3975_1004    conda-forge
xorg-libxrender           0.9.10            h14c3975_1002    conda-forge
xorg-libxt                1.1.5             h14c3975_1002    conda-forge
xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
xorg-xextproto            7.3.0             h14c3975_1002    conda-forge
xorg-xproto               7.0.31            h14c3975_1007    conda-forge
xz                        5.2.4             h14c3975_1001    conda-forge
yaml                      0.1.7             h14c3975_1001    conda-forge
zeromq                    4.2.5             hf484d3e_1006    conda-forge
zict                      0.1.3                      py_0    conda-forge
zlib                      1.2.11            h14c3975_1004    conda-forge
zope.interface            4.6.0           py36h14c3975_1000    conda-forge


```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : prod
    active env location : /opt/miniconda3/envs/prod
            shell level : 1
       user config file : /home/akrherz/.condarc
 populated config files : /home/akrherz/.condarc
          conda version : 4.5.12
    conda-build version : not installed
         python version : 3.6.7.final.0
       base environment : /opt/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/pypi/linux-64
                          https://conda.anaconda.org/pypi/noarch
                          https://conda.anaconda.org/HCC/linux-64
                          https://conda.anaconda.org/HCC/noarch
                          https://conda.anaconda.org/cgat/linux-64
                          https://conda.anaconda.org/cgat/noarch
          package cache : /opt/miniconda3/pkgs
                          /home/akrherz/.conda/pkgs
       envs directories : /opt/miniconda3/envs
                          /home/akrherz/.conda/envs
               platform : linux-64
             user-agent : conda/4.5.12 requests/2.21.0 CPython/3.6.7 Linux/3.10.0-957.1.3.el7.x86_64 rhel/7.6 glibc/2.17
                UID:GID : 1000:1000
             netrc file : /home/akrherz/.netrc
           offline mode : False
```
</details>

If I do an isolated install of fiona, it pulls in gdal=2.4.0, whereas my env has gdal=2.3.2-py36hb00a9d7_0

**Comment by ocefpaf** [comment]
> If I do an isolated install of fiona, it pulls in gdal=2.4.0, whereas my env has gdal=2.3.2-py36hb00a9d7_0

How are you creating that env? We have `fiona` built with `gdal 2.3.3` and `gdal 2.4` that works, I wonder why your env is pulling 2.3.2 there :-/

**Comment by akrherz** [comment]
> How are you creating that env? We have `fiona` built with `gdal 2.3.3` and `gdal 2.4` that works, I wonder why your env is pulling 2.3.2 there :-/

I have convoluted ways :(  Those versions are blocked
```
$ conda install gdal=2.4.0
Solving environment: failed

UnsatisfiableError: The following specifications were found to be in conflict:
  - esmf
  - gdal=2.4.0
Use "conda info <package>" to see the dependencies for each package.

$ conda install gdal=2.3.3
Solving environment: failed

UnsatisfiableError: The following specifications were found to be in conflict:
  - descartes
  - gdal=2.3.3
Use "conda info <package>" to see the dependencies for each package.
```

**Comment by ocefpaf** [comment]
`esmf` was not migrated to the new compilers and "unsafe" to be  installed with the old compilers `label`. (That s probably the source of your problems. Pinging @mariusvniekerk here who suggested we remove the old compiler packages from `main`.)

Try to use `--channel conda-forge/label/cf201901` in your command invocation or substitute `conda-forge` for `conda-forge/label/cf201901` in your `.condarc`.

**Comment by akrherz** [comment]
Thanks @ocefpaf , I tried the `--channel conda-forge/label/cf201901`, but after an hour of `conda` spinning its wheels with 'Solving environment:', I gave up.

**Comment by ocefpaf** [comment]
> Thanks @ocefpaf , I tried the `--channel conda-forge/label/cf201901`, but after an hour of `conda` spinning its wheels with 'Solving environment:', I gave up.

Yeah, that has all of conda-forge in there and it is slow. However, it should not take 1-hour though :-/ Something else may be amiss there. (I have a bad connection here on South America and it took only a few minutes for me.)

BTW, have you tried metachannel? It is a temporary solution until conda solver gets faster that one of the conda-forge core members (@mariusvniekerk) created. See https://github.com/regro/conda-metachannel for more info.

**Comment by akrherz** [comment]
> BTW, have you tried metachannel?

I have not and thanks for the continued excellent help and ideas here.  I ended up restoring from backup and am wondering if I filed this issue in the wrong place.  My understanding is some baseline library updated built from a newer compiler and my environment contains an older `gdal` built against the older compiler.  So I close this issue and only similiar ones in `descartes` and `esmf`, which are blocking my current need up upgrade `gdal`?
